selfservice/flow/verification/handler.go

```diff
@@ -123,6 +123,19 @@ func (h *Handler) NewVerificationFlow(w http.ResponseWriter, r *http.Request, ft
 	return f, nil
 }
 
+// Create Verification Flow Parameters for Native Apps
+//
+// swagger:parameters createNativeVerificationFlow
+//
+//nolint:deadcode,unused
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type createNativeVerificationFlow struct {
+	// The URL to return after the flow was completed.
+	//
+	// in: query
```

Comment:
Does this actually work? Native flows don’t use redirects afaik?

Reply:
Not redirections _per se_, but this piece of information is contained in the Verification Flow.

For instance calling `http://127.0.0.1:4433/self-service/verification/api?return_to=http://127.0.0.1:4433/ok` would returns this flow content:
```
{
    "id": "689e20e6-f66c-4bcb-996c-d1f3c87e7670",
    "type": "api",
    "expires_at": "2024-09-19T14:52:43.434168067Z",
    "issued_at": "2024-09-19T13:52:43.434168067Z",
    "request_url": "http://127.0.0.1:4433/self-service/verification/api?return_to=http://127.0.0.1:4433/ok",
    "return_to": "http://127.0.0.1:4433/ok",
    "active": "code",
[...]
```

With a `return_to` key, which we can be used in our logic to know the redirection URL set at the beginning of the flow :)

Reply:
I rephrased the sentence.
Let me know what you think :)